src/app/reducers/firebaseUserReducer.js

```diff
@@ -0,0 +1,24 @@
+import * as ActionTypes from '../actions/types';
+
+export default function (state = null, action) {
+  switch (action.type) {
```

Comment:
Are we ever going to want to return anything other than action.payload? Because, if not, we could simplify this a lot